rpms/s3/s3rpm.spec

```diff
@@ -170,15 +170,15 @@ rm -rf %{buildroot}
 ./installhelper.sh %{buildroot} --release
 # Install the background delete python module.
 cd %{_builddir}/%{name}-%{version}-%{_s3_git_ver}/s3backgrounddelete
-python%{py_ver} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT
+python%{py_ver} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --version=%{version}
```

Comment:
Check at line 158. We are doing duplicate echos of message "build complete".

Remove one of the echo cmd.